app/models/katello/concerns/organization_extensions.rb

```diff
@@ -32,8 +32,9 @@ module OrganizationExtensions
         has_many :org_tasks, :dependent => :destroy, :class_name => "Katello::TaskStatus", :inverse_of => :organization
 
         attr_accessor :statistics
+        attr_accessible :label
 
-        scope :having_name_or_label, ->(name_or_label) { { :conditions => ["name = :id or label = :id", {:id => name_or_label}] } }
+        scope :having_name_or_label, ->(name_or_label) { where("name = :id or label = :id", { :id => name_or_label } )
```

Comment:
Kind of a different problem, if you want I can create a separate Redmine issue for it. The previous block was not executing anything, the way to scope things on Rails 4 is by running something in the block argument that actually scopes the model, not just providing the conditions.